mode/verilog/verilog.js

```diff
@@ -125,6 +137,24 @@ CodeMirror.defineMode("verilog", function(config, parserConfig) {
     if (ch == '`') {
       stream.next();
       if (stream.eatWhile(/[\w\$_]/)) {
+        var cur = stream.current();
+        curKeyword = cur;
+        // Macros that end in _begin, are start of block and end with _end
+        if (cur.startsWith("`uvm_") && cur.endsWith("_begin")) {
+          var keywordClose = curKeyword.substr(0,curKeyword.length - 5) + "end";
+          openClose[cur] = keywordClose;
+          curPunc = "newblock";
+        } else if (cur.startsWith("`uvm_") && cur.endsWith("_end")) {
+        } else {
```

Comment:
Hi, what happened here? Did you forget something?

Reply:
Hi,
Good question! ...I left it in because I didn't want that hitting the "else" branch (should have put a comment or something there...actually, maybe I should have just negated the logic and made the "else" into an "else if (!(...))".

That said, now that you bring it up and I've slept, the behavior would be just fine with line 147 removed.

Should I make a new pull request and remove that line?

Reply:
> Should I make a new pull request and remove that line?

Feel free to. It might save someone some confusion in the future.

Reply:
I have made PR #6517